File: lua/refactoring/tests/refactor/106/c/prompt_func_param_return/extract.expected.c
#include <stdio.h>

int foo(int a) {
  int i = 3;
  printf("%d %d", i, a);
  return i;
}


int simple_function(int a) {
  printf("this is a test\n");
  INSERT_TYPE_HERE i = foo(a);


  return i;
}

File: lua/refactoring/tests/refactor/106/java/prompt_func_param_return/extract.expected.java
class Extract {

public static int foo(int a) {
        System.out.println("this is a test");
        int test = 1;
        System.out.println(a);
        return test;
}


    public static int simpleFunction(int a) {
        var test = foo(a);

        System.out.println(test);

        return test;
    }
}
